hazelcast/src/main/java/com/hazelcast/scheduledexecutor/IScheduledExecutorService.java

```diff
@@ -0,0 +1,416 @@
+/*
+ * Copyright (c) 2008-2016, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.scheduledexecutor;
+
+import com.hazelcast.core.DistributedObject;
+import com.hazelcast.core.Member;
+import com.hazelcast.spi.annotation.Beta;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.Callable;
+import java.util.concurrent.RejectedExecutionException;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Distributed & durable implementation of {@link ScheduledExecutorService}.
```

Comment:
> Distributed & durable implementation of {@link ScheduledExecutorService}.

`IScheduledExecutorService ` is not extending `ScheduledExecutorService` hence this is not an implementation of it. Doc may be a left over?